components/HomeHeader.vue

```diff
@@ -1,16 +1,19 @@
 <template>
   <swiper
-    v-if="items.length > 0 && !$vuetify.breakpoint.mobile"
+    v-if="items.length > 0 && !$vuetify.breakpoint.mobile && loaded"
+    ref="homeSwiper"
     class="swiper"
     :options="swiperOptions"
+    @slideChange="onSlideChange"
   >
     <swiper-slide v-for="item in items" :key="item.Id">
-      <div
-        class="slide-backdrop"
-        :style="{
-          backgroundImage: `url('${getBackdrop(item)}')`
-        }"
-      />
+      <div class="slide-backdrop">
+        <blurhash-image
+          :key="`${item.Id}-${reloadSentinel}`"
+          :item="getRelatedItem(item)"
+          :type="'Backdrop'"
+        />
```

Comment:
```suggestion
        <blurhash-image
          v-if="getRelatedItem(item)"
          :key="`${item.Id}-${reloadSentinel}`"
          :item="getRelatedItem(item)"
          :type="'Backdrop'"
        />
```

This fixes my `this.item cannot be undefined` error, but then some items don't have a backdrop at all:
![image](https://user-images.githubusercontent.com/19396809/102552362-61e20400-40c1-11eb-9187-f070bcb159c8.png)

Reply:
@Mrtimscampi Now I see the problem.

I only return items that have related items in the function, so items that must display its own background images are completely skipped.

I'm going to push a fix, but it doesn't fully solves the issue, as we don't do any filtering in the initial items to check that everything will display a photo, so items without metadata will still be displayed totally black.

Reply:
>  but it doesn't fully solves the issue, as we don't do any filtering in the initial items to check that everything will display a photo

That's not an issue :) We should show the latest, no matter if it has a backdrop or not. But these had a backdrop and worked before, hence the bug :yum: